### DISCUSSION TITLE
Error en Reloj Cotalker - Android/ iOS / Versión tienda y App no concuerdan.

### DISCUSSION THREAD

**Discussion by soporte-cotalker** [Cotalker/documentation#483] [4. Bug Reports] [answered]
Estimados, según lo conversado, en plataforma móviles nuevamente no permite ingresar los tiempos con detalle al minuto

Interfaz sólo permite incrementos de 5 min.

Versión de app en tienda y web no coinciden.

![image](https://user-images.githubusercontent.com/51834544/137756448-84129b09-e592-40dc-acb1-c0e8f1a3141b.png)

![image](https://user-images.githubusercontent.com/51834544/137756489-bfeed40c-0c0f-4857-89dc-7aec7c21b471.png)

**Comment by jdurandpaz** [comment]
se crea issue https://github.com/Cotalker/documentation/issues/493

**Comment by guillevalin** [comment, marked as answer]
Este incidente fue resuelto en la última versión del aplicativo iOS, el cual ya está en la tienda. Sin embargo, como se informa en el issue #493 se trabajará en una mejora de la funcionalidad para mejorar su usabilidad.
